webapp-django/crashstats/crashstats/static/crashstats/js/socorro/timeutils.js

```diff
@@ -0,0 +1,24 @@
+/* global moment */
+
+$(function() {
+    function updateTimes() {
+        $('time.ago').each(function() {
+            var self = $(this);
+            var date = self.attr('datetime') || self.data('date');
+            self
+                .attr('title', self.text())
+                .text(moment(date).fromNow());
+        });
+
+        $('time.in').each(function() {
+            var self = $(this);
+            var date = self.attr('datetime') || self.data('date');
+            self
+                .attr('title', self.text())
+                .text(moment(date).fromNow(true));
+        });
+    }
+
+    updateTimes();
+    setInterval(updateTimes, 60 * 1000);
+});
```

Comment:
I'm totally confused. There's a `timeutils.js` in https://github.com/mozilla/socorro/pull/3420/files already. But in this PR it looks like a new file.